$OpenBSD: patch-src_media_library_cpp,v 1.1 2011/09/23 09:45:18 dcoppa Exp $

Clear filters before jumping to media library: fix "Jumping to
song...Aborted" crash (git commit 3f064df95dc0cd57176429894791f199b376379f)

--- src/media_library.cpp.orig	Thu Apr 21 02:31:02 2011
+++ src/media_library.cpp	Fri Sep 23 11:20:27 2011
@@ -633,6 +633,7 @@ void MediaLibrary::LocateSong(const MPD::Song &s)
 	
 	if (!hasTwoColumns)
 	{
+		Artists->ApplyFilter("");
 		if (Artists->Empty())
 			Update();
 		if (primary_tag != Artists->Current())
@@ -650,6 +651,7 @@ void MediaLibrary::LocateSong(const MPD::Song &s)
 		}
 	}
 	
+	Albums->ApplyFilter("");
 	if (Albums->Empty())
 		Update();
 	
@@ -672,6 +674,7 @@ void MediaLibrary::LocateSong(const MPD::Song &s)
 		}
 	}
 	
+	Songs->ApplyFilter("");
 	if (Songs->Empty())
 		Update();
 	
